MasterPassword/C/mpw-cli.c

```diff
@@ -210,21 +210,26 @@ int main(int argc, char *const argv[]) {
         masterPassword = getpass( "Your master password: " );
 
     // Summarize operation.
-    fprintf( stderr, "%s's password for %s:\n[ %s ]: ", fullName, siteName, mpw_identicon( fullName, masterPassword ) );
+    char const * identicon = mpw_identicon( fullName, masterPassword );
+    fprintf( stderr, "%s's password for %s:\n[ %s ]: ", fullName, siteName, identicon );
+	mpw_freeString( identicon );
 
     // Output the password.
     const uint8_t *masterKey = mpw_masterKeyForUser(
             fullName, masterPassword, algorithmVersion );
     mpw_freeString( masterPassword );
+    mpw_freeString( fullName );
```

Comment:
I'm not convinced it's a good idea to claim ownership of and subsequently free `fullName`.  I can see making a case for doing so, but then we need to make sure we claim ownership over it in all places, including `getenv` etc.  Prior to your change, is `fullName` currently leaking in any way?